components/lib/colorpicker/ColorPicker.js

```diff
@@ -97,13 +97,21 @@ export const ColorPicker = React.memo(
             !isUnstyled && DomHandler.addClass(elementRef.current, 'p-colorpicker-dragging');
         };
 
+        const getPositionY = (event) => {
+            if (event.pageY !== undefined) return event.pageY;
+            else if (event.touches !== undefined) return event.touches[0].pageY;
```

Comment:
should this be `touches` or `changedTouches` ?

Reply:
OOPS!! you're right 
thank you for comment :) ([63881b0](https://github.com/primefaces/primereact/pull/6788/commits/63881b09ff842091d4921d7d0747747b4ca58147))